files/jquery.prettycheckable/update.json

```diff
@@ -0,0 +1,8 @@
+{
+  "packageManager": "github",
+  "name": "prettyCheckable",
+  "repo": "arthurgouveia/prettyCheckable",
+  "files": {
+    "include": ["./img/*", "./dist/*", "./dev/*"]
```

Comment:
So you'd remove the "./" from "./dist"?

The problem is, `prettyCheckable.css` has a reference to the image inside this using `background:url('../img/sprites-sfa68604977.png')`.

So as far I understand, this would not work anymore.

Btw, if removing "./dev/*", then we don't even need all the other images.

So maybe we can add `"exclude": ["./img/sprites/*"]` ?